annotation_tracker/web_client/views/ImageView/index.js

```diff
@@ -0,0 +1,27 @@
+import ImageView from '@girder/histomicsui/views/body/ImageView';
+import { wrap } from '@girder/core/utilities/PluginUtils';
+import _ from 'underscore';
+
+import Experiments from '../../panels/Experiments';
+
+wrap(ImageView, 'initialize', function (initialize) {
+  this.experiments = new Experiments({
+    parentView: this,
+  });
+
+  initialize.apply(this, _.rest(arguments));
+});
+
+wrap(ImageView, 'render', function (render) {
+  render.call(this);
+
+  this.$('.h-control-panel-container')
+    .removeClass('hidden')
+    .append('<div class="h-experiment-widget s-panel"></div>');
```

Comment:
We probably want to wrap this command with `if (!this.$('.h-experiment-widget').length)` so we don't add another div every time render is called.

Reply:
To support moving and changing panels, please add `id="h-experiment-panel"` to this div.

Reply:
See 46c4fb0ba7e1a1add4d830c43f34f8454eb1d41e and 272901434a2b38c7ead9cd72bb1c0455d51bf612.